kernel/src/device/pty/driver.rs

```diff
@@ -41,6 +47,13 @@ impl PtyDriver {
 
         let mut output = self.output.lock();
         if output.is_empty() {
+            if self.opened_slaves.load(Ordering::Relaxed) == 0 {
+                return_errno_with_message!(
+                    Errno::EIO,
+                    "the pty master does not have opened slaves"
+                );
+            }
+
```

Comment:
Prefer not to have this blank line because it's related to `if output.is_empty() {`?